src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -1206,18 +1111,14 @@ public String toString() {
     // next highest count total (validates that we were correctly batch eliminated)
     final BigDecimal nextHighestTally;
 
-    // function: BatchElimination constructor
-    // purpose: create a new BatchElimination object simple container
-    // param: candidateId the candidate eliminated
-    // param: runningTotal how many total votes were totaled when this candidate was eliminated
-    // param: nextHighestTally next highest count total
-    // returns: the new object
     BatchElimination(String candidateId, BigDecimal runningTotal, BigDecimal nextHighestTally) {
       this.candidateId = candidateId;
       this.runningTotal = runningTotal;
       this.nextHighestTally = nextHighestTally;
     }
   }
 
-  static class TabulationCancelledException extends Exception {}
+  static class TabulationCancelledException extends Exception {
```

Comment:
why?

Reply:
again, it's what intelliJ -- presumably form the google code style jar does